README.md

```diff
@@ -46,6 +46,15 @@ which includes:
 1. `drush deploy`
 1. Generating a login link.
 
+### `robo deploy:drupal`
+
+The `deploy:drupal` command (often available in downstream repos as
+`composer robo deploy:drupal SITENAME DOCROOT`) deploys Drupal in a standardized way
+which includes:
+
+1. `drush deploy`
+1. `drush config:import`
```

Comment:
Good question. That's a can of worms that is documented in the command itself. 😬 

- https://github.com/ChromaticHQ/usher/blob/2.x/src/Robo/Plugin/Commands/DeployCommands.php#L68

How would you suggest I denote this to avoid future confusion while avoiding duplication?

Add something like `See the code for additional information`?